---
layout: page
title: Synergistically Developed Resources  
date: 2019-07-06T13:19:48+09:00
modified: 2020-02-19T12:24:48+09:00
excerpt: "Synergistic Activities"
tags: [R, statistics, workshops, teaching]
image:
  feature: image-1.jpg
  fig-caption: Gunnison Point, Black Canyon of Gunnison
  credit: Arun Seetharam
  creditlink: https://aseetharam.github.io
---


<section id="table-of-contents" class="toc">
  <header>
    <h3>Overview</h3>
  </header>
<div id="drawer" markdown="1">
*  Auto generated table of contents
{:toc}
</div>
</section><!-- /#table-of-contents -->

## Educational

### Bioinformatics Workbook

Bioinformatics and biological data analysis are an important aspect of biological research. Researchers in the genomic era ask larger questions that routinely require dealing with large amounts of data that require familiarity with medium to large scale computing. We created a GitHub based bioinformatics workbook [bioinformaticsworkbook.org](https://bioinformaticsworkbook.org/), that can serve as an online knowledge repository for scientists. This workbook covers variety of problems, with the walk-through examples.

**Website** [bioinformaticsworkbook.org](https://bioinformaticsworkbook.org/)

_If you use the workbook for your research, please cite us:_

Arun Seetharam, Siva Chudalayandi, Rick Masonbrink, Usha Muppirala, Adam Rivers, Maryam Sayadi, & Andrew Severin. (2019, October 11). ISUgenomics/bioinformatics-workbook: 2019-10-11 Release of the Bioinformatics Workbook (Version v1.0.0). _Zenodo_ [![DOI](https://zenodo.org/badge/91881720.svg)](https://zenodo.org/badge/latestdoi/91881720) [<img src="https://img.shields.io/badge/-Source-blue?labelColor=black&style=flat&logo=github"/>](https://github.com/ISUgenomics/bioinformatics-workbook)


### Workshop on Basic UNIX

This exercise is designed to provide the basic skills required for working in the UNIX environment, using plenty of relevant examples, specifically for biologists.  If you are using your personal computer, make sure that you have downloaded the files required for the workshop. This exercise will provide you information regarding navigation, files and directory creation/modification and some administrative things related to file permissions.
[<img src="https://img.shields.io/badge/-Slides-blue?labelColor=white&style=flat&logo=Microsoft-PowerPoint"/>](/publications/pdf/slides_workshop_as.pptx)
[<img src="https://img.shields.io/badge/-Exercises-blue?labelColor=white&style=flat&logo=Adobe-Acrobat-Reader"/>](/publications/pdf/UNIX_exercises_all.pdf)
[<img src="https://img.shields.io/badge/-Materials-blue?labelColor=black&style=flat&logo=github"/>](https://github.com/ISUgenomics/basic_UNIX_2015)

### Workshop on Phylogenomics

This phylogenomics tutorial is divided into 3 major sections. The first section deals with identification of
orthologs from closely related plasmodium species. Second section is about multiple sequence alignment
and construction of super matrix. The final section will show simple phylogenetic tree construction using
the Maximum Likelihood and Bayesian analyses.
[<img src="https://img.shields.io/badge/-VMware%20(3.6Gb)-blue?labelColor=white&style=flat"/>](/publications/pdf/)
[<img src="https://img.shields.io/badge/-Exercises-blue?labelColor=white&style=flat&logo=Adobe-Acrobat-Reader"/>](/publications/pdf/phylogenomics_workshop.pdf)
[<img src="https://img.shields.io/badge/-Materials-blue?labelColor=black&style=flat&logo=github"/>](https://github.com/aseetharam/phylogenetics)

## Web resources


**[NAM-genomes.org](https://nam-genomes.org/)**: is consortium website containing information regarding the maize NAM founder lines genome assembly and annotation. The website includes links to raw data and final results distributed via [MaizeGDB](https://maizegdb.org/NAM_project). The genomes have been made public on Jan 9th, 2020. [<img src="https://img.shields.io/badge/go_to-NAM_genomes-green?labelColor=black&style=flat"/>](https://nam-genomes.org)


**[GenomeQC](https://genomeqc.maizegdb.org/)** is a user-friendly and interactive platform that generates descriptive summaries with intuitive graphics for genome assemblies and structural annotations. It also benchmarks user supplied assemblies and annotations against the publicly available reference genomes of their choice.
[<img src="https://img.shields.io/badge/Try-GenomeQC-yellowgreen?labelColor=black&style=flat"/>](https://genomeqc.maizegdb.org/)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/HuffordLab/GenomeQC)

**[SeriolaDB](https://www.serioladb.org/)** is a website designed as a centralized knowledge base for Seriola researchers and producers.  It contains genomics, genetics, breeding and phenotypic data along with protocols and tools to translate this information in to practical applications for culture activities.  SeriolaDB uses GMOD Tripal and Drupal's luggage extension to manage the genomics data and empower efficient web development by multiple users. [<img src="https://img.shields.io/badge/go_to-SeriolaDB-green?labelColor=black&style=flat"/>](https://www.serioladb.org/)

**[SCNbase](https://scnbase.org)** is a website designed as a centralized knowledge base for Soybean Cyst Nematode (SCN) researchers and producers.  It contains genomics, genetics, population data along tools to translate this information in to practical applications.  SCNBase uses GMOD Tripal and Drupal's luggage extension to manage the genomics data and empower efficient web development by multiple users. [<img src="https://img.shields.io/badge/go_to-SCNbase-green?labelColor=black&style=flat"/>](https://scnbase.org/)


## Software

**[pyrpipe](https://github.com/urmi-21/pyrpipe)**: is python package to easily develop RNA-Seq analyses workflow by integrating popular RNA-Seq analysis programs in an object oriented manner.[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/urmi-21/pyrpipe)

**[SequelTools](https://github.com/ISUgenomics/SequelTools)**:  is a fast and easy to install command-line program that provides a collection of tools for working with multiple SMRTcells of BAM format PacBio Sequel raw sequence data. [<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/ISUgenomics/SequelTools)

**[phylostratr](https://github.com/arendsee/phylostratr)**: A fully automated R framework predict and explore the age of genes using phylostratigraphic methods. The program selects species for balanced representation, retrieving sequences, building databases, inferring phylostrata and returning diagnostics. [<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/arendsee/phylostratr)

**[fagin](https://github.com/arendsee/fagin)**: a synteny-based pipeline in the R framework. fagin determines the genomic context of each query gene in a focal species compared to homologous sequence in target species. [<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/arendsee/fagin)
