---
layout: page
title: Publications
date: 202-02-29T19:09:34+01:00
modified: 202-02-29T19:09:34+01:00
excerpt: "Arun Seetharam's publications"
tags: [Arun Seetharam, Arun S. Seetharam]
image:
  feature: image-3.jpg
  credit: Arun Seetharam
  creditlink: https://aseetharam.github.io
---

<span id='badgeCont419579' style='width:126px'>
<script src='http://labs.researcherid.com/mashlets?el=badgeCont419579&mashlet=badge&showTitle=false&className=a&rid=B-8899-2015'>
</script>
</span>
<section id="table-of-contents" class="toc">
  <header>
    <h3>Overview</h3>
  </header>
<div id="drawer" markdown="1">
*  Auto generated table of contents
{:toc}
</div>
</section><!-- /#table-of-contents -->

## Preprints/In-Press
<!-- pubs: 5; tot: 49 -->

- Jonathan Cahn, Michael Regulski, Jason Lynn, Evan Ernst, Cristiane de Santis Alves, Srividya Ramakrishnan, Kapeel Chougule, Sharon Wei, Zhenyuan Lu, Xiaosa Xu, Jorg Drenkow, Melissa Kramer, **Arun S. Seetharam**, Matthew B. Hufford, W. Richard McCombie, Doreen Ware, David Jackson, Michael C. Schatz, Thomas R. Gingeras, and Robert A. Martienssen.
MaizeCODE reveals bi-directionally expressed enhancers that harbor molecular signatures of maize domestication.
_bioRxiv_,
10.1101/2024.02.22.581585, 2024
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/2024.02.22.581585)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/martienssenlab/maize-code)

 
- Carson M. Andorf, Jeffrey Ross-Ibarra, Arun S. Seetharam, Matthew B. Hufford, and Margaret R. Woodhouse,
A unified VCF data set from nearly 1,500 diverse maize accessions and resources to explore the genomic landscape of maize,
_bioRxiv_, 10.1101/2024.04.30.591904, 2024
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/2024.04.30.591904)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/Maize-Genetics-and-Genomics-Database/SNPversity2.0)
[<img src="https://img.shields.io/badge/SNPversity2.0-yellow?style=flat"/>](https://wgs.maizegdb.org/)

- Julia Engelhorn, Samantha J. Snodgrass, Amelie Kok, **Arun S. Seetharam**, Michael Schneider, Tatjana Kiwit, Ayush Singh, Michael Banf, Merritt Khaipho-Burch, Daniel E. Runcie, Victor Sánchez Camargo, J. Vladimir Torres-Rodriguez, Guangchao Sun, Maike Stam, Fabio Fiorani, James C. Schnable, Hank W. Bass, Matthew B. Hufford, Benjamin Stich, Wolf B. Frommer, Jeffrey Ross-Ibarra, and Thomas Hartwig.
Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize.
_bioRxiv_, 10.1101/2023.08.08.551183, 2023
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/2023.08.08.551183)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/Snodgras/MOA_Analysis)

- Aimee J Schulz, Jingjing Zhai, Taylor AuBuchon-Elder, Mohamed El-Walid, Taylor H Ferebee, Elizabeth H Gilmore, Matthew B Hufford, Lynn C Johnson, Elizabeth A Kellogg, Thuy La, Evan Long, Zachary R Miller, M Cinta Romay, **Arun S. Seetharam**, Michelle C Stitzer, Travis Wrightsman, Edward S Buckler, Brandon Monier, Sheng-Kai Hsu.
Fishing for a reelGene: evaluating gene models with evolution and machine learning.
_bioRxiv_,
10.1101/2023.09.19.558246, 2023
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/2023.09.19.558246)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>]()

- Travis Wrightsman, Taylor H. Ferebee, M. Cinta Romay, Arun S. Seetharam, Taylor AuBuchon-Elder, Alyssa R. Phillips, Michael Syring, Matthew B. Hufford, Elizabeth A. Kellogg, and Edward S. Buckler,
Current genomic deep learning architectures generalize across grass species but not alleles.
_bioRxiv_,
10.1101/2024.04.11.589024, 2024,
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/2024.04.11.589024)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>]()


- Urminder Singh, Jeffrey A. Haltom, Joseph W. Guarnieri, Jing Li, Arun Seetharam, Afshin Beheshti, Bruce Aronow, and Eve Syrkin Wurtele
A pan-tissue, pan-disease compendium of human orphan genes
_bioRxiv_,
10.1101/2024.02.21.581488, 2024
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/2024.02.21.581488)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/jahaltom/Orphan-Gene-Supp)


## 2024

<!-- pubs: 3; tot: 44 -->

- Shujun Ou, Tyler Collins, Yinjie Qiu, **Arun S. Seetharam**, Claire C. Menard, Nancy Manchanda, Jonathan I Gent, Michael C. Schatz, Sarah. N. Anderson, Matthew B Hufford, and Candice N Hirsch.
Differences in activity and stability drive transposable element variation in tropical and temperate maize. _Genome Research_, 34(8) 10.1101/gr.278131.123, 2024
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/gr.278131.123 )
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/oushujun)


- Jessica A. Kinkade, **Arun S. Seetharam**, Shrikesh Sachdev, Nathan J. Bivens, Brett S. Phinney, Gabriela Grigorean, R Michael Roberts, Geetu Tuteja, Cheryl S. Rosenfeld, Extracellular vesicles from mouse trophoblast cells: effects on neural progenitor cells and potential participants in the placenta-brain Axis, _Biology of Reproduction_, ioad146, 2023.
[<img src="https://img.shields.io/badge/-Artcile-lightgrey?style=flat"/>](https://doi.org/10.1093/biolre/ioad146)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://tuteja-lab.github.io/mouse.trophoblast.smallRNAseq)


- Foster Tyler L., Monika Kloiber-Maitz, Laurine Gilles, Ursula K. Frei, Sarah Pfeffer, Yu-Ru Chen, Somak Dutta, Arun S. Seetharam, Matthew B. Hufford, and Thomas Lübberstedt,
Fine mapping of major QTL qshgd1 for spontaneous haploid genome doubling in maize (_Zea mays_ L.).
_Theoretical and Applied Genetics_ 137, no. 5
10.1007/s00122-024-04615-y
[<img src="https://img.shields.io/badge/-Artcile-lightgrey?style=flat"/>](https://doi.org/10.1007/s00122-024-04615-y)


## 2023
<!-- pubs: 3; tot: 41 -->


- Silas Tittes, Anne Lorant, Sean McGinty, James B. Holland, Jose de Jesus Sánchez-González, **Arun S. Seetharam**, Maud Tenaillon, and Jeffrey Ross-Ibarra,
Not so local: the population genetics of convergent adaptation in maize and teosinte,
 _eLife_, 12:RP92405
10.7554/eLife.92405.1.sa2, 2023
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.7554/eLife.92405.1.sa2)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/silastittes/parv_local)


- Thomas Hartwig, Michael Banf, Gisele Passaia Prietsch, Jia-Ying Zhu, Isabel Mora-Ramírez, Jos H. M. Schippers, Samantha J. Snodgrass, Arun S. Seetharam, Bruno Huettel, Judith M. Kolkman, Jinliang Yang, Julia Engelhorn, Zhi-Yong Wang, Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize
_Genome Biology_ 24(108) 2023.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1186/s13059-023-02909-w)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/mbanf/HaschSeq)


- Alyssa R. Phillips, **Arun S. Seetharam**, Taylor AuBuchon-Elder, Edward S. Buckler, Lynn J. Gillespie, Matthew B. Hufford, Victor Llaca, M. Cinta Romay, Robert J. Soreng, Elizabeth A. Kellogg, Jeffrey Ross-Ibarra. A happy accident: a novel turfgrass reference genome, _G3_, 10.1101/2022.03.08.483531, 2023
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1101/2021.09.09.459637)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/silastittes/parv_local)


## 2022
<!-- pubs: 1; tot: 38 -->
- **Arun S. Seetharam**, Ha T.H. Vu, Sehee Choi, Teka Khan, Megan A. Sheridan, Toshihiko Ezashi, R. Michael Roberts, Geetu Tuteja, The product of BMP-directed differentiation protocols for human primed pluripotent stem cells is placental trophoblast and not amnion, _Stem Cell Reports_, 17(6) p1289-1302, 2022.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1016/j.stemcr.2022.04.014)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://tuteja-lab.github.io/BAPvsTrophoblast_Amnion/)


## 2021
<!-- pubs: 8; tot: 37 -->

- Jing Li, Urminder Singh, Priyanka Bhandary, Jacqueline Campbell, Zebulun Arendsee, **Arun S. Seetharam**, Eve Syrkin Wurtele.Foster thy young: Enhanced prediction of orphan genes in assembled genomes,  _Nucleic Acids Research_, gkab1238_, 2021.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1093/nar/gkab1238)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/eswlab/orphan-prediction)

- **Arun Seetharam**<sup>&#8224;</sup>, Yunqing Yu<sup>&#8224;</sup>, Sébastien Belanger, Lynn G. Clark, Blake C. Meyers, Elizabeth A. Kellogg, Matthew B. Hufford.
The _Streptochaeta_ genome and the evolution of the grasses.
_Frontiers in Plant Science_, vol. 12, p2165, 2021.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.3389/fpls.2021.710383)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/HuffordLab/streptochaeta)


- Matthew B Hufford, **Arun S Seetharam**, Margaret R Woodhouse, Kapeel M. Chougule, Shujun Ou, Jianing Liu, William A Ricci, Tingting Guo, Andrew Olson, Yinjie Qiu, Rafael Della Coletta, Silas Tittes, Asher I Hudson, Alexandre P. Marand, Sharon Wei, Zhenyuan Lu, Bo Wang, Marcela K Tello-Ruiz, Rebecca D Piri, Na Wang, Dong won Kim, Yibing Zeng, Christine H O'Connor, Xianran Li, Amanda M Gilbert, Erin Baggs, Ksenia V Krasileva, John L Portwood II, Ethalinda KS Cannon, Carson M Andorf, Nancy Manchanda, Samantha J Snodgrass, David E Hufnagel, Qiuhan Jiang, Sarah Pedersen, Michael L Syring, David A Kudrna, Victor Llaca, Kevin Fengler, Robert J. Schmitz, Jeffrey Ross-Ibarra, Jianming Yu, Jonathan I Gent, Candice N Hirsch, Doreen Ware, R Kelly Dawe.
_De novo_ assembly, annotation, and comparative analysis of 26 diverse maize genomes.
_Science_, Vol. 373, Issue 6555, pp. 655-662, 2021. [<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1126/science.abg5289) [<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/HuffordLab/NAM-genomes)


- Teka Khan<sup>&#8224;</sup>,  **Arun S. Seetharam**<sup>&#8224;</sup>, Jie Zhou, Nathan J. Bivens, Danny J. Schust, Toshihiko Ezashi, Geetu Tuteja and R M. Roberts.
Single nucleus RNA sequence (snRNAseq) analysis of the spectrum of trophoblast lineages generated from human pluripotent stem cells in vitro.
_Frontiers in Cell and Developmental Biology_, vol. 9, p1881, 2021.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.3389/fcell.2021.695248)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/Tuteja-Lab/BAP.hESC.d8_snRNAseq)

- Nolan Bornowski, Kathryn J. Michel, John P. Hamilton, Shujun Ou, **Arun S. Seetharam**, Jerry Jenkins, Jane Grimwood, Chris Plott, Shengqiang Shu, Jayson Talag, Megan Kennedy, Hope Hundley, Vasanth R. Singan, Kerrie Barry, Chris Daum, Yuko Yoshinaga, Jeremy Schmutz, Candice N. Hirsch, Matthew B. Hufford, Natalia de Leon, Shawn M. Kaeppler, C. Robin Buell.
Genomic variation within the maize Stiff Stalk heterotic germplasm pool.
_The Plant Genome_, e20114, 2021.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1002/tpg2.20114)

- Urminder Singh, Jing Li, **Arun S Seetharam** and Eve S Wurtele. pyrpipe: a Python package for RNA-Seq workfows. _NAR Genomics and Bioinformatics_, 3(2) lqab04, 2021.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1093/nargab/lqab049)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/urmi-21/pyrpipe)

- Wyatt A. Shell, Michael A. Steffen, Hannah K. Pare, **Arun S. Seetharam**, Andrew J. Severin, Amy L. Toth and Sandra M. Rehan.
Sociality sculpts similar patterns of molecular evolution in two independently evolved lineages of eusocial bees.
_Communications Biology_  (4) 253, 2021
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1038/s42003-021-01770-6)

- Karthik Murugan, Shravanti Suresh,  **Arun S Seetharam**, Andrew J Severin, and Dipali G Sashital.
Systematic in vitro specificity profiling reveals nicking defects in natural and engineered CRISPR-Cas9 variants.
 _Nucleic Acids Research_, gkab163, 2021.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1093/nar/gkab163)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/sashital-lab/Cas9_specificity)

## 2020
<!-- pubs: 6; tot: 29 -->
- David E Hufnagel, Matthew B Hufford, and **Arun S Seetharam**.
SequelTools: a suite of tools for working with PacBio Sequel raw sequence data.
_BMC Bioinformatics_, (21) 429, 2020.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1186/s12859-020-03751-8)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/ISUgenomics/SequelTools)

- Jianing Liu, **Arun S Seetharam**, Kapeel Chougule, Shujun Ou, Kyle W. Swentowsky, Jonathan I. Gent, Victor Llaca, Margaret R. Woodhouse, Nancy Manchanda, Gernot G. Presting, David A. Kudrna, Magdy Alabady, Candice N. Hirsch, Kevin A. Fengler, Doreen Ware, Todd P. Michael, Matthew B. Hufford and R. Kelly Dawe,
Gapless assembly of maize chromosomes using long read technologies.
_Genome Biology_, (21) 121, 2020.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1186/s13059-020-02029-9)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/dawelab/Ab10-Assembly)

- Shujun Ou, Jianing Liu, Kapeel M. Chougule, Arkarachai Fungtammasan, **Arun Seetharam**, Joshua Stein, Victor Llaca, Nancy Manchanda, Amanda M. Gilbert, Xuehong Wei, Chen-Shan Chin, David E. Hufnagel, Sarah Pedersen, Samantha Snodgrass, Kevin Fengler, Margaret Woodhouse, Brian P. Walenz, Sergey Koren, Adam M. Phillippy, Brett Hannigan, R. Kelly Dawe, Candice N. Hirsch, Matthew B. Hufford, Doreen Ware.
Effect of sequence depth and length in long-read assembly of the maize inbred NC358.
_Nature Communications_ 11(2288), 2020
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1038/s41467-020-16037-7)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/HuffordLab/Maize_NC358)

- Karthik Murugan, **Arun S Seetharam**, Andrew J Severin, and Dipali G Sashital.
CRISPR-Cas12a has widespread off-target and dsDNA-nicking effects.
_Journal of Biological Chemistry_, jbc-RA120, 2020.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1074/jbc.RA120.012933)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/ISUgenomics/sashital-lab/tree/master/2019_cas12aNickase)

- Nancy Manchanda, John L Portwood, Margaret R Woodhouse, **Arun S Seetharam**, Carolyn J Lawrence-Dill, Carson M Andorf, and Matthew B Hufford.
GenomeQC: A quality assessment tool for genome assemblies and gene structure annotations.
_BMC Genomics_ 21(193), 2020.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1186/s12864-020-6568-2)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/HuffordLab/GenomeQC)
[<img src="https://img.shields.io/badge/GenomeQC-yellow?style=flat"/>](https://genomeqc.maizegdb.org/)

- Abdelnaser M Elashry, Samer S Habash, Paramasivan Vijayapalani, Nahal Brocke-Ahmadinejad, Roman Blümel, **Arun Seetharam**, Heiko Schoof, and Florian MW Grundler.
Transcriptome and parasitome analysis of Beet Cyst Nematode _Heterodera schachtii_.
_Scientific Reports_, 10(1):1--12, 2020.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1038/s41598-020-60186-0)


## 2019
<!-- pubs: 6; tot: 23 -->
- Rick E Masonbrink, Catherine M Purcell, Sara E Boles, Andrew Whitehead, John R Hyde, **Arun S Seetharam**, and Andrew J Severin.
An annotated genome for _Haliotis rufescens_ (red abalone) and resequenced green, pink, pinto, black, and white abalone species.
_Genome biology and evolution_, 11(2):431--438, 2019.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1093/gbe/evz006)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/ISUgenomics/RedAbaloneGenomePaper_GBE_2018)

- Rick E Masonbrink, Tom R. Maier, Usha Muppirala, **Arun S. Seetharam**, Etienne Lord, Parijat S. Juvale, Jeremy Schmutz, Nathan T. Johnson, Dmitry Korkin, Melissa G. Mitchum, Benjamin Mimee, Sebastian Eves-van den Akker, Matthew Hudson, Andrew J. Severin and Thomas J. Baum .
The genome of the soybean cyst nematode (_Heterodera glycines_) reveals complex patterns of duplications involved in the evolution of parasitism genes.
_BMC genomics_, 20(1):119, 2019.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1186/s12864-019-5485-8)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/remkv6/SCN_Genome_Paper)

- Stacey N Barnes, Rick E Masonbrink, Thomas R Maier, **Arun Seetharam**, Anoop S Sindhu, Andrew J Severin, and Thomas J Baum.
_Heterodera glycines_ utilizes promiscuous spliced leaders and demonstrates a unique preference for a species-specific spliced leader over _C. elegans_ SL1.
_Scientific reports_, 9(1):1--11, 2019.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1038/s41598-018-37857-0)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/ISUgenomics/Heterodera-glycines-Spliced-Leaders)

- Zebulun Arendsee, Jing Li, Urminder Singh, Priyanka Bhandary, **Arun Seetharam**, and Eve Syrkin Wurtele.
fagin: synteny-based phylostratigraphy and finer classification of young genes.
_BMC bioinformatics_, 20(1):1--14, 2019.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1186/s12859-019-3023-y)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/arendsee/fagin)

- Zebulun Arendsee, Jing Li, Urminder Singh, **Arun Seetharam**, Karin Dorman, and Eve Syrkin Wurtele. phylostratr: A framework for phylostratigraphy.
_Bioinformatics_, 35(19):3617--3627, 2019.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1093/bioinformatics/btz171)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/arendsee/phylostratr)

- Rick Masonbrink, Tom R Maier, **Arun S Seetharam**, Parijat S Juvale, Levi Baber, Thomas J Baum, and Andrew J Severin. SCNbase: a genomics portal for the Soybean Cyst Nematode (_Heterodera glycines_).
_Database_, 2019(1):baz111, 2019.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1093/database/baz111)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/remkv6/SCNBase)
[<img src="https://img.shields.io/badge/go_to-SCNbase-green?labelColor=black&style=flat"/>](https://scnbase.org/)

## 2018
<!-- pubs: 3; tot: 17 -->
- Priyanka Bhandary, **Arun S Seetharam**, Zebulun W Arendsee, Manhoi Hur, and Eve Syrkin Wurtele. Raising orphans from a metadata morass: A researcher's guide to re-use of public’omics data. _Plant science_, 267:32--47, 2018.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1016/j.plantsci.2017.10.014)

- Catherine M. Purcell, **Arun S. Seetharam**, Owyn Snodgrass, Sofia Ortega-García, John R. Hyde, and Andrew J. Severin. Insights into teleost sex determination from the _Seriola dorsalis_ genome assembly. _BMC Genomics_, 19(31), 2018.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1186/s12864-017-4403-1)
[<img src="https://img.shields.io/badge/go_to-SeriolaDB-green?labelColor=black&style=flat"/>](https://www.serioladb.org/)

- Joshua T Trujillo, **Arun S Seetharam**, Matthew B Hufford, Mark A Beilstein, and Rebecca A Mosher. Evidence for a unique DNA-dependent RNA polymerase in cereal crops. _Molecular biology and evolution_, 35(10):2454--2462, 2018.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1093/molbev/msy146)


## 2017
<!-- pubs: 4; tot: 14 -->
- Mingfeng Cao, Meirong Gao, Carmen Lorena Lopez-Garcia, Yutong Wu, **Arun S. Seetharam**, Andrew Josef Severin, and Zengyi Shao.
Centromeric DNA facilitates nonconventional yeast genetic engineering.
_ACS synthetic biology_, 6(8):1545--1553, 2017.
[<img src="https://img.shields.io/badge/-Artcile-lightgrey?style=flat"/>](https://doi.org/10.1021/acssynbio.7b00046)

- J. N. Cullen, A. Lithio, **A. S. Seetharam**, Y. Zheng, G. Li, D. Nettleton, and A.M. O’Connor.
Microbial community sequencing analysis of the calf eye microbiota and relationship to infectious bovine kerato-conjunctivitis.
_Veterinary Microbiology_, 207(2017):267--279, 2017.
[<img src="https://img.shields.io/badge/-Artcile-lightgrey?style=flat"/>](https://doi.org/10.1016/j.vetmic.2017.07.003)

- Mingfeng Cao, **Arun S. Seetharam**, Andrew J. Severin, and Zengyi Shao.
Rapid isolation of centromeres from _Scheffersomyces stipitis_.
_ACS synthetic biology_, 6(11):2028--2034, 2017.
[<img src="https://img.shields.io/badge/-Artcile-lightgrey?style=flat"/>](https://doi.org/10.1021/acssynbio.7b00166)

- Rick E Masonbrink, Andrew J Severin, and **Arun S Seetharam**.
Comparative genomics of soybean and other legumes.
In _The Soybean Genome_, pages 83--93. Springer, 2017.
[<img src="https://img.shields.io/badge/-Get_Chapter-lightgrey?style=flat"/>](https://link.springer.com/book/10.1007/978-3-319-64198-0)

## 2016
<!-- pubs: 1; tot: 10 -->
- Basudev Chowdhury<sup>&#8224;</sup>, **Arun Seetharam**<sup>&#8224;</sup>, Zhiping Wang, Yunlong Liu, Amy C. Lossie, Jyothi Thimmapuram, and Joseph Irudayaraj.
A study of alterations in DNA epigenetic modifications (5mC and 5hmC) and gene expression influenced by simulated microgravity in human lymphoblastoid cells.
_PLoS ONE_, 11(1):e0147514, 2016.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>]( https://doi.org/10.1371/journal.pone.0147514)


## 2015
<!-- pubs: 3; tot: 9 -->
- **Arun S Seetharam**, Emily Kawaler, Zhi-Qiang Du, Max F Rothschild, and Andrew J Severin.
Microbiome analyses of pacific white shrimp (_Litopenaeus vannamei_) collected from disparate geographical locations.
_Genomics data_, 6:67, 2015.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1016/j.gdata.2015.08.009)

- Chaoyou Xue, **Arun S Seetharam**, Olga Musharova, Konstantin Severinov, Stan J J. Brouns, Andrew J Severin, and Dipali G Sashital.
CRISPR interference and priming varies with individual spacer sequences.
_Nucleic acids research_, 43(22):10831--10847, 2015.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1093/nar/gkv1259)

- **Arun Seetharam**, Antonio Gomez, Catherine M Purcell, John R Hyde, Philip D Blood, and Andrew J Severin.
NCBI-BLAST programs optimization on XSEDE resources for sustainable aquaculture.
In _Proceedings of the 2015 XSEDE Conference: Scientific Advancements Enabled by Enhanced Cyberinfrastructure_, pages 1--5, 2015.
[<img src="https://img.shields.io/badge/-Artcile-lightgrey?style=flat"/>](https://doi.org/10.1145/2792745.2792749)

## Ph.D/Masters
<!-- pubs: 6; tot: 6 -->
- **Arun S Seetharam** and Gary W Stuart.
Whole genome phylogeny for 21 _Drosophila_ species using predicted 2b-RAD fragments.
_PeerJ_, 1:e226, 2013.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.7717/peerj.226)
[<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github"/>](https://github.com/aseetharam/phylogenomic_profiler)

- **Arun Seetharam** and Gary W Stuart.
A study on the distribution of 37 well conserved families of C2H2 zinc finger genes in eukaryotes.
_BMC Genomics_, 14(1):420, 2013.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1186/1471-2164-14-420)

- Rajkumar, B. Fakrudin, S. P. Kavil, Y. Girma, **S. S. Arun**, D. Dadakhalandar, B. H. Gurusiddesh, A. M. Patil, M. Thudi, S. B. Bhairappanavar.
Molecular mapping of genomic regions harbouring QTLs for root and yield traits in sorghum (_Sorghum bicolor_ L. Moench).
_Physiology and Molecular Biology of Plants_, 19(3):409--419, 2013. [<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://dx.doi.org/10.1007%2Fs12298-013-0188-0)

- **Arun Seetharam** and Gary W Stuart.
Whole genome phylogenies for multiple _Drosophila_ species.
_BMC research notes_, 5(1):670, 2012.
[<img src="https://img.shields.io/badge/-Open_Access-blue?style=flat&logo=Open-Access"/>](https://bmcresnotes.biomedcentral.com/articles/10.1186/1756-0500-5-670#citeas)

- **Arun Seetharam**, Yang Bai, and Gary W Stuart.
A survey of well conserved families of C2H2 zinc-finger genes in _Daphnia_.
_BMC genomics_, 11(1):276, 2010.
[<img src="https://img.shields.io/badge/Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1186/1471-2164-11-276)

- P. Srinivasa Reddy, B. Fakrudin, Rajkumar, S. M. Punnuri, **S. S. Arun**, M. S. Kuruvinashetti, I. K. Das, and N. Seetharama.
Molecular mapping of genomic regions harboring QTLs for stalk rot resistance in sorghum.
_Euphytica_, 159(1--2):191--198, 2007.
[<img src="https://img.shields.io/badge/Open_Access-blue?style=flat&logo=Open-Access"/>](https://doi.org/10.1007/s10681-007-9472-9)


## Thesis/Dissertation
<!-- pubs: 0; tot: 0 -->
-  **Arun Somwarpet Seetharam.**
_Phylogenomics: Molecular Evolution in the Genomics Era_.
PhD thesis, 2012.
[<img src="https://img.shields.io/badge/get_pdf-gray?style=flat&logo=Adobe-Acrobat-Reader"/>](/publications/pdf/arun-seetharam-phd-thesis.pdf)

- **SS Arun.**
_In Silico Est Datamining for Elucidation of Repeats Biology and Functional Annotation in Sorghum_ [_Sorghum bicolor_ (L.) Moench.].
Masters thesis, UAS, Dharwad, 2006.
[<img src="https://img.shields.io/badge/get_pdf-gray?style=flat&logo=Adobe-Acrobat-Reader"/>](/publications/pdf/arun-seetharam-msc-thesis.pdf)



<sup>&#8224;</sup> _shared first authors_
